src/interface/src/app/treatments/treatments.state.ts

```diff
@@ -52,7 +53,16 @@ export class TreatmentsState {
   private _activeProjectArea$ =
     new BehaviorSubject<TreatmentProjectArea | null>(null);
```

Comment:
I assume we really need to update _activeProjectArea$, but the change so far doesn't appear to break anything.